### DISCUSSION TITLE
UserWarning: You have unused kwarg parameters in Row, please remove them: {'scale': 1}

### DISCUSSION THREAD

**Discussion by devnakx** [anapnoe/stable-diffusion-webui-ux#119] [Q&A] [answered]
`stable-diffusion-webui-ux\venv\lib\site-packages\gradio\deprecation.py:43: UserWarning: You have unused kwarg parameters in Row, please remove them: {'scale': 1}`
`stable-diffusion-webui-ux\venv\lib\site-packages\gradio\components.py:4417: UserWarning: The 'grid' parameter will be deprecated. Please use 'columns' instead.`
I just updated to the latest commit but received this warning during startup.
Although it runs well and has no errors, will this warning be fixed?
I don't think it's a bug, so I didn't post this in issues.

**Comment by anapnoe** [comment]
thank you it has been fixed

**Reply by devnakx** [reply]
thank you very much, but I received new warnings:
`stable-diffusion-webui-ux\venv\lib\site-packages\gradio\deprecation.py:43: UserWarning: You have unused kwarg parameters in Row, please remove them: {'equal_height': True}
  warnings.warn(`
`stable-diffusion-webui-ux\venv\lib\site-packages\gradio\deprecation.py:43: UserWarning: You have unused kwarg parameters in Tab, please remove them: {'open': False}
  warnings.warn(`
you can see it with checked `Show warnings in console.` in `Settings` - `System`
I can uncheck it to disable warnings, but I think it let you know would be better

**Reply by anapnoe** [reply, marked as answer]
ok thanks I will fix them too gradio change a ton of things yes I know just thought that you posted all warnings related to gradio but it seems they are thrown in chunks ;)

**Reply by anapnoe** [reply]
also some extensions may introduce the issue if they haven't been updated

**Reply by devnakx** [reply]
> also some extensions may introduce the issue if they haven't been updated

okay i see
